### PR TITLE
Update AlchemySubprovider with the most recent alchemy-web3 package 

### PR DESCRIPTION
**Pull request by Kharabet**
https://app.clubhouse.io/bzx1/story/1189/update-alchemysubprovider-with-the-most-recent-alchemy-web3

### CODE REVIEW

File: packages/alchemy-web3/.gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ node_modules
 yarn-debug.log
 yarn-error.log
 
+dist
```
Comment (AoDev) [inline packages/alchemy-web3/.gitignore:9]: Wont that prevent us from updating it next time since we store the dist in the repo?

Comment (Kharabet) [reply packages/alchemy-web3/.gitignore:9]: The dist folder is removed on package build. So next time we will update alchemy-web3, the dist folder will be updated with 
 already transpiled code as well.

**Comment by Kharabet** [comment]
We will not use AlchemySubprovider for now. The RPCSubprovider should be used as data-fetching provider at PR #373. But I would like to update alchemy-web3 package anyway.

**Comment by Kharabet** [comment]
AlchemySubprovider is not used currently for readonly RPC calls.

**Pull request merged**